src/main/java/Deadline.java

```diff
@@ -0,0 +1,16 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    private LocalDate deadline;
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-M-dd");
+    Deadline(String name, String deadLine) {
```

Comment:
Perhaps deadline in the constructor parameter be written as "deadline" instead of deadLine.